sinks/blackhole/README.md

```diff
@@ -0,0 +1,11 @@
+# Blackhole Sink
+
+This sink sends Veneur metrics to nowhere.
+
+# Status
+
+**This sink is for testing and benchmark purposes**. We're unsure why else you wan't to use this.
+
+# Configuration
+
+Can only be added by manually inserting it into the server's sinks.
```

Comment:
I love the READMEs! I think we should move this section in all the docs here up towards the beginning - it's arguably the most important piece of information (: